cfpb/consumerfinance.gov
# Consolidate breadcrumb templates

## Changes

- Consolidate breadcrumb templates under `molecules/breadcrumb.html` template.
- Convert redundant breadcrumb blocks to `breadcrumb_items ` data structures.
- Adds breadcrumbs to wellbeing survey pages.


## How to test this PR

The following pages should have proper breadcrumbs when compared to production:

http://localhost:8000/about-us/diversity-and-inclusion/voluntary-assessment-onboarding-form/
http://localhost:8000/about-us/diversity-and-inclusion/voluntary-assessment-onboarding-form/form-submitted/
http://localhost:8000/about-us/diversity-and-inclusion/voluntary-assessment-onboarding-form/privacy-act-statement/

http://localhost:8000/data-research/hmda/historic-data/

http://localhost:8000/ask-cfpb/what-is-a-credit-reporting-company-en-1251/
http://localhost:8000/ask-cfpb/search/?q=test

http://localhost:8000/about-us/blog/borrowing-or-lending-money-to-family-or-friends-use-this-worksheet/
http://localhost:8000/compliance/supervision-examinations/compliance-management-review-examination-procedures/
http://localhost:8000/enforcement/actions/capital-one-bank/

http://localhost:8000/about-us/events/archive-past-events/2021-release-of-the-taskforce-on-federal-consumer-financial-law-report/
http://localhost:8000/data-research/

http://localhost:8000/compliance/compliance-resources/consumer-lending-resources/payday-lending-rule/payday-lending-rule-faqs/
http://localhost:8000/compliance/
http://localhost:8000/your-story/

http://localhost:8000/about-us/careers/current-openings/examiner-commissioned-compliance-examiner-21-cfpb-158-mpa-/

http://localhost:8000/rules-policy/regulations/

http://localhost:8000/consumer-tools/educator-tools/youth-financial-education/teach/activities/
http://localhost:8000/consumer-tools/educator-tools/youth-financial-education/teach/activities/sonya-gives-advice/

http://localhost:8000/data-research/prepaid-accounts/search-agreements/detail/72669/

http://localhost:8000/consumer-tools/financial-well-being/results/
http://localhost:8000/consumer-tools/financial-well-being/results/es/

http://localhost:8000/consumer-tools/financial-well-being/error/
http://localhost:8000/consumer-tools/financial-well-being/error/es/
http://localhost:8000/consumer-tools/financial-well-being/about/
http://localhost:8000/consumer-tools/financial-well-being/about/es/
